unidist/core/backends/mpi/core/executor.py

```diff
@@ -18,6 +18,7 @@
 
 # MPI stuff
 comm, rank, world_size = communication.get_mpi_state()
+topology = dict()
```

Comment:
Initialization to dict is safer then `None` in Python?

Reply:
Not sure I understand, what you mean by "safer". We can initialize a global variable with any value . `dict()` is more applicable in this case because implementation of `executor.cluster_resources()` assumes that `topology` is `dict` (in case if `topology` won't be initialized by something reason we got empty `dict` on API layer instead of error).